Dockerfile

```diff
@@ -12,25 +14,30 @@ RUN apt-get update && \
     gnupg \
     gcc
 
+
+RUN echo "ENVIRONMENT VAR:  $MSSQL_REQUIRED"
+
 # SQL Server (MS SQL)
 # https://docs.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-microsoft-odbc-driver-for-sql-server?view=sql-server-ver15
-RUN apt-get install apt-transport-https
-RUN curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add -
-RUN curl https://packages.microsoft.com/config/debian/10/prod.list | tee /etc/apt/sources.list.d/msprod.list
-RUN apt-get update
+RUN if [ "$MSSQL_REQUIRED" = "true" ] ; then apt-get install apt-transport-https ; fi
+RUN if [ "$MSSQL_REQUIRED" = "true" ] ; then curl https://packages.microsoft.com/keys/microsoft.asc | apt-key add - ; fi
+RUN if [ "$MSSQL_REQUIRED" = "true" ] ; then curl https://packages.microsoft.com/config/debian/10/prod.list | tee /etc/apt/sources.list.d/msprod.list ; fi
+RUN if [ "$MSSQL_REQUIRED" = "true" ] ; then apt-get update ; fi
 ENV ACCEPT_EULA=y DEBIAN_FRONTEND=noninteractive
-RUN apt-get -y install \
+RUN if [ "$MSSQL_REQUIRED" = "true" ] ; then apt-get -y install \
     unixodbc-dev \
     msodbcsql17 \
-    mssql-tools
+    mssql-tools ; fi
 
 # Update pip and install requirements
-COPY requirements.txt dev-requirements.txt ./
+COPY requirements.txt dev-requirements.txt mssql-requirements.txt ./
 RUN pip install -U pip  \
     && pip install 'cryptography~=3.4.8' \
     && pip install snowflake-connector-python --no-use-pep517  \
     && pip install -r requirements.txt -r dev-requirements.txt
```

Comment:
We'll want to change this too such that we're not installing `dev-requirements.txt` in a production context